src/index.html

```diff
@@ -7,8 +7,52 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+      type="text/css"
+    >
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="navigationBar">
+      <a href="#home" class="nav_link">
+        <img src="images/logo.png" alt="icon of the website">
+      </a>
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a href="https://" class="is-active nav_link">Apple</a>
```

Comment:
These links now lead to a non existent site empty site, it would be better for them to lead to ids based on their values, here it would be -> `href="#apple"`